packages/esm-patient-common-lib/src/types/test-results.ts

```diff
@@ -39,3 +39,10 @@ export type OBSERVATION_INTERPRETATION =
   | 'LOW'
   | 'CRITICALLY_LOW'
   | 'OFF_SCALE_LOW';
+
+export interface ExternalOverviewProps {
+  patientUuid: string;
+  filter: (filterProps: PanelFilterProps) => boolean;
+}
+
+export type PanelFilterProps = [entry: ObsRecord, uuid: string, type: string, panelName: string];
```

Comment:
Either make an object / interface here that has properties / names or drop the (otherwise invisible) labels.

My suggestion:

```
export type PanelFilterProps = [ObsRecord, string, string, string];
```

Reply:
Done